src/python_testing/TC_HSTAT_Test_Base.py

```
@@ -0,0 +1,101 @@
+from matter.interaction_model import InteractionModelError, Status
+
+log = logging.getLogger(__name__)
+
+
+class HSTATBase:
+    """
+    This is a base class for Humidistat cluster tests.
+    """
+
+    async def setup(self):
+        self.endpoint = self.get_endpoint()
```

Pylint (4.0.6)

[error] Instance of 'HSTATBase' has no 'get_endpoint' member

(E1101)